src/partials/_head.hbs

```diff
@@ -9,4 +9,5 @@
 
 	grunticon(["/icons/icons.data.svg.css", "/icons/icons.data.png.css", "/icons/icons.fallback.css"], grunticon.svgLoadedCallback);
 </script>
+<script src="/dist/index.js"></script>
```

Comment:
I don't think I'm loading any jQuery. My javascript actually isn't even working right now, so would moving the tag down the page fix that?